src/App.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { getMissions } from './redux/missions/missionSlice';
+import { getRockets } from './redux/rockets/rocketsSlice';
+import Router from './Router';
+import Navbar from './components/Navbar';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissions());
+    dispatch(getRockets());
+  }, [dispatch]);
```

Comment:
- [ ] When I start up the application both the missions and the rocket details are fetched but that should not be so. When I start up the application only the rocket details should be fetched (since it takes me to the rockets page). Only when I navigate to the missions page should the missions be fetched. Kindly fix that.

  You can use the `redux-logger` package to inspect your store in the browser

Reply:
Hi @julie-ify, thanks a lot for your review.